modules/data.atmosphere/R/closest_xy.R

```diff
@@ -37,13 +40,35 @@ closest_xy <- function(slat, slon, infolder, infile) {
         }
       }
     }
+  } else {
+    ## this case may make assumptions specific to PalEON Regional
```

Comment:
not sure what the Paleon lat/lon will look like here but if they are vectors, i.e. `dim(lat) = NULL`, wouldn't they meet `all(dim(lat) == dim(lon))` condition above? `all(NULL == NULL)` returns `TRUE` I think